src/Mails_PT.php

```diff
@@ -177,4 +206,44 @@ private function addRecipientGroupFields( string $group, $cmb ) {
 		) );
 	}
 
+	//Create a custom HTML cmb Field for All variables
+	public function custom_variables_field($cmb, $variables)
+	{
+		$cmb->add_field(array(
+			'name' => 'Mail Variables',
+			'id'   => self::POST_TYPE_NAME . "mail_variables",
+			'type' => 'text',
+			'variables' => $variables,
+			'render_row_cb' => function ($field_args, $field) {
+				$id          = $field->args('id');
+				$label       = $field->args('name');
+				$variables = $field->args('variables');
+
+?>
+			<div class="custom-field-row">
+				<h3><label for="<?php echo $id; ?>"><?php echo $label; ?></label></h3>
+				<br>
+				<?php
+				foreach ($variables as $key => $value) {
+					echo '<h5 class="cmb2-metabox-title" ">' . $key . '</h5><br>';
+					if (is_array($value) || is_object($value)) {
+						foreach ($value as $objectkey => $objectvalue) {
+							if (!empty($objectvalue) && !is_array($objectvalue) && !is_object($objectvalue)) {
+								echo "<div class='juvo_tooltip'>  <span class='juvo_tooltiptext'>{$objectvalue}</span> <a  href='#'  class='juvo_variable_badges'>{{{$objectkey}}}</a></div>";
+							}
+						}
+					} else {
+						echo "<div class='juvo_tooltip'>  <span class='juvo_tooltiptext'>{$value}</span> <a  href='#' class='juvo_variable_badges'>{{{$key}}}</a></div>";
+					}
+				}
+
+				?>
+
+				<input id='badgesInput' value='' type='hidden'>;
+			</div>
+<?php
+			},
+		));
+	}
```

Comment:
Move this code in a custom twig layout in /admin/view/placeholder-widget.twig. In the callback of the cmb2 box display property, use Timber::compile() to output the layout.